maven-plugin-report-plugin/src/it/plugin-report/verify.groovy

```diff
@@ -28,3 +28,11 @@ assert !pluginInfo.text.contains('Disk Space')
 // check JDK and Maven requirements
 assert pluginInfo.text.contains('1.8')
 assert pluginInfo.text.contains('3.2.5')
+
+assert new File( basedir, 'target/site/de/noop-mojo.html' ).isFile()
+assert new File( basedir, 'target/site/de/report-mojo.html' ).isFile()
+assert new File( basedir, 'target/site/de/plugin-info.html' ).isFile()
+
+assert new File( basedir, 'target/site/fr/noop-mojo.html' ).isFile()
+assert new File( basedir, 'target/site/fr/report-mojo.html' ).isFile()
+assert new File( basedir, 'target/site/fr/plugin-info.html' ).isFile()
```

Comment:
Should we also check `en` locale and default?

Reply:
This is already done in the script: https://github.com/apache/maven-plugin-tools/pull/215/files/4b9556f99f5335cf0f4ebba3e151f8c0157ea7d8#diff-e6a05d5a0be5ba47cd4a603f6422fb5f8467085d97800b02bc82b60f9582853dR23.